ipalib/install/certmonger.py

```diff
@@ -156,10 +156,11 @@ def __init__(self):
                                           DBUS_CM_IF)
 
 
-def _get_requests(criteria=dict()):
+def _get_requests(criteria=None):
```

Comment:
I guess why have a default at all? Why not just _get_request(criteria) and enforce that criteria is dict?

Since you're poking at it probably worth documenting that:

:param criteria: dict of the search criteria

Reply:
Yep, i just removed the default arg, because it is always called with arg.